blogposts/2020/evolution-of-the-precise-code-intel-backend.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Evolution of the precise code intel backend"
+description: This post reflects on the high-level technical changes as the precise code intel services providing the feature matured through additional features, changing environment requirements, hardening, performance improvements, refactoring, and one major rewrite in a different language. 
+author: Eric Fritz
+authorUrl: https://eric-fritz.com
+publishDate: 2020-06-17T10:00-07:00
+tags: [blog]
+slug: evolution-of-the-precise-code-intel-backend
+heroImage: /blog/big-brain.png
+published: true
+---
+
+On July 12, Sourcegraph's LSIF-based precise code intelligence will have received its first commit a year ago.
+
+<div class="text-center">
+    <strong>🎉 Happy 🎉 Almost 🎉 Birthday 🎉</strong>
+</div>
+
+Sourcegraph's [precise code intelligence features](https://docs.sourcegraph.com/user/code_intelligence/lsif) are driven by user-uploaded LSIF indexes created in their own build and continuous integration systems. When browsing code that has been indexed, all hover tooltips, definition, and reference results are _precise_ rather than heuristic (based off of search results, which is the no-configuration default).
```

Comment:
```suggestion
Sourcegraph's [precise code intelligence features](https://docs.sourcegraph.com/user/code_intelligence/lsif) are driven by user-uploaded LSIF indexes created in their own build and continuous integration systems. When browsing code that has been indexed, all hover tooltips, definitions, and reference results are _precise_ rather than heuristic (based off of search results, which is the no-configuration default).
```